### PR TITLE
Fixed heading for benefits section

### PR DESCRIPTION
**Pull request by ghost**
@openshift/team-documentation Please peer review.

### CODE REVIEW

File: scaling_performance/optimizing_storage.adoc [hunk 1]
```diff
@@ -145,7 +145,9 @@ Corruption may occur when using NFS to back {production-title} scaled/HA registr
 In an {product-title} hosted metrics cluster deployment:
 
 * The preferred storage technology is block storage.
-* It is not recommended to use NAS storage (excluding CNS/CRS GlusterFS as it uses a block storage interface from iSCSI) for hosted metrics cluster deployment with production workloads.
+* It is not recommended to use NAS storage (excluding CNS/CRS GlusterFS as it uses
+a block storage interface from iSCSI) for hosted metrics cluster deployment with
+production workloads.
```
Comment (mburke5678) [inline scaling_performance/optimizing_storage.adoc]: Something seems missing here:
for **a** hosted metrics cluster deployment
for hosted metrics cluster deployment**s**

File: scaling_performance/optimizing_storage.adoc [hunk 2]
```diff
@@ -158,7 +160,9 @@ Corruption may occur when using NFS to back hosted metrics at production workloa
 In an {product-title} hosted logging cluster deployment:
 
 * The preferred storage technology is block storage.
-* It is not recommended to use NAS storage (excluding CNS/CRS GlusterFS as it uses a block storage interface from iSCSI) for hosted metrics cluster deployment with production workloads.
+* It is not recommended to use NAS storage (excluding CNS/CRS GlusterFS as it
+ uses a block storage interface from iSCSI) for hosted metrics cluster
+ deployment with production workloads.
```
Comment (mburke5678) [inline scaling_performance/optimizing_storage.adoc]: Same as previous

File: scaling_performance/optimizing_storage.adoc [hunk 3]
```diff
@@ -186,10 +190,12 @@ to nodes to support a healthy cluster.
 === Choosing a Docker Graph Driver
 
 Docker stores images and containers in a graph driver (a pluggable storage
-technology), such as `DeviceMapper`, `Overlay`, and `Btrfs`. Each have advantages and
-disadvantages. For example, `Overlay` is faster than `DeviceMapper` at starting and
-stopping containers, but is not Portable Operating System Interface for Unix (POSIX) compliant because of the architectural
-limitations of a union file system, and does not yet support SELinux.
+technology), such as `DeviceMapper`, `Overlay`, and `Btrfs`. Each have
+advantages and disadvantages. For example, `Overlay` is faster than
```
Comment (mburke5678) [inline scaling_performance/optimizing_storage.adoc]: s/Each have/Each has

**Comment by mburke5678** [comment]
@tmorriso-rh A couple nits. Otherwise LGTM

**Comment by tmorriso-rh** [comment]
Duplicate of https://github.com/openshift/openshift-docs/pull/6511

**Pull request closed without merging**